app/controllers/idv/opt_in_ipp_controller.rb

```diff
@@ -0,0 +1,93 @@
+module Idv
+  class OptInIppController < ApplicationController
+    # include AcuantConcern
+    # include DocumentCaptureConcern
+    include IdvStepConcern
+    include StepIndicatorConcern
+    include PhoneQuestionAbTestConcern
+
+    # todo: do we need rate limiting here?
+    # before_action :confirm_not_rate_limited
+    # before_action :confirm_in_person_needed
+    # before_action :override_csp_to_allow_acuant
+    # todo: add a check_or_render_not_found guard
+
+    def show
+      analytics.idv_doc_auth_document_capture_visited(**analytics_arguments)
+
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('document_capture', :view, true)
+
+      render :show, locals: extra_view_variables
+    end
+
+    def update
+      idv_session.redo_document_capture = nil # done with this redo
+      # Not used in standard flow, here for data consistency with hybrid flow.
+      document_capture_session.confirm_ocr
+
+      result = handle_stored_result
+      analytics.idv_doc_auth_document_capture_submitted(**result.to_h.merge(analytics_arguments))
+
+      Funnel::DocAuth::RegisterStep.new(current_user.id, sp_session[:issuer]).
+        call('document_capture', :update, true)
```

Comment:
todo: update this with a new step for this page

Reply:
I'm not sure we're updating the db table for this funnel anymore. @zachmargolis ?

Reply:
we're still producing reports based on it, until we drop the table entirely, I'd suggest we keep writing to it

Reply:
correct, yes, no new columns